spec/results/azure.yml

```diff
@@ -192,33 +229,36 @@ instance_groups:
         bosh:
           blobstores:
           - options:
-              endpoint: http://10.99.0.25:25250
+              endpoint: https://10.99.0.25:25250
               password: <!{meta.vault}/blobstore/agent:password!>
               tls:
                 cert:
                   ca: <!{meta.vault}/blobstore/ca:certificate!>
-              user: blobstore-user
+              user: agent
             provider: dav
-      mbus: nats://natsuser:<!{meta.vault}/nats:password!>@10.99.0.25:4222
+          ntp:
+          - 0.pool.ntp.org
+          - 1.pool.ntp.org
+      mbus: nats://nats:<!{meta.vault}/nats:password!>@10.99.0.25:4222
     azure:
       client_id: <!{meta.vault}/azure:client_id!>
-      client_secret: <!{meta.vault}/azure:client_secret!>
+      client_secret: (( vault meta.vault "/azure:client_secret" )
       default_security_group: test-default-sg
       environment: AzureCloud
       resource_group_name: test-resource-group
       ssh_public_key: <!{meta.vault}/azure/ssh:public!>
       ssh_user: vcap
+      storage_account_name: test-storage_account_name
       subscription_id: <!{meta.vault}/azure:subscription_id!>
-      tenant_id: <!{meta.vault}/azure:tenant_id!>
-      use_managed_disks: true
```

Comment:
https://github.com/cloudfoundry/bosh-azure-cpi-release/blob/master/jobs/azure_cpi/spec#L58
is by default false

Reply:
Lets use this file by default for azure: https://github.com/genesis-community/bosh-genesis-kit/blob/ccf46010baf4b69cd0d7509ea04d57aecff4d6d7/bosh-deployment/azure/use-managed-disks.yml, there is no reason to not use managed disks. And since it was enabled by the kit by default we don't have to deal with migrations

Reply:
add this to blueprint by default